deepface/modules/preprocessing.py

```diff
@@ -33,6 +35,10 @@ def load_image(img: Union[str, np.ndarray]) -> Tuple[np.ndarray, str]:
     # The image is a base64 string
     if img.startswith("data:image/"):
         return load_base64(img), "base64 encoded string"
+    
+    # The video is a base64 string
+    if img.startswith("data:video/"):
+        return loadVideo_base64(img), "base64 encoded string"   
```

Comment:
load_image's return type is numpy array or string, but load video base 64 is returning list of string.